test/e2e/framework.go

```diff
@@ -637,6 +637,19 @@ func (data *TestData) mutateFlowAggregatorConfigMap(ipfixCollector string, faClu
 	return nil
 }
 
+// deleteFlowAggregator deploys flow aggregator with ipfix collector address.
```

Comment:
it seems that you typically use "Flow Aggregator" and not "flow aggregator" in comments
also, technically it should be "the Flow Aggregator"